generated/niscope/niscope/_grpc_stub_interpreter.py

```diff
@@ -252,7 +252,7 @@ def fetch_into_numpy(self, channel_list, timeout, num_samples):  # noqa: N802
     def fetch_array_measurement(self, channel_list, timeout, array_meas_function, measurement_waveform_size):  # noqa: N802
         response = self._invoke(
             self._client.FetchArrayMeasurement,
-            grpc_types.FetchArrayMeasurementRequest(vi=self._vi, channel_list=channel_list, timeout=timeout, array_meas_function_raw=array_meas_function.value),
+            grpc_types.FetchArrayMeasurementRequest(vi=self._vi, channel_list=channel_list, timeout=timeout, array_meas_function_raw=array_meas_function.value, meas_wfm_size=measurement_waveform_size),
```

Comment:
Do older versions of the server ignore this field? This will likely cause the server to return a different waveform size than what you asked for.

Reply:
>Do older versions of the server ignore this field? This will likely cause the server to return a different waveform size than what you asked for.
That's what my testing indicated, yes.

With an older ni_grpc_device_server.exe:
>FAILED test_system_niscope.py::TestGrpc::test_fetch_array_measurement[less_than_actual] - assert 2000 == 1000

Reply:
I think mentioning it in the changelog should be sufficient, based on how we handled the corresponding issue in LabVIEW:

https://www.ni.com/en-us/support/documentation/bugs/23/measurementlink-2023-q1-known-issues.html